src/components/v5/common/CompletedAction/partials/PaymentBuilder/partials/FundingModal/types.ts

```diff
@@ -0,0 +1,13 @@
+import { type ExpenditurePayout } from '~gql';
+import { type Expenditure } from '~types/graphql.ts';
+import { type ModalProps } from '~v5/shared/Modal/types.ts';
+
+export type TokenItemProps = Omit<ExpenditurePayout, 'isClaimed'>;
+
+export interface FundingItem extends TokenItemProps {
+  key: number;
+}
```

Comment:
I might be missing something but "token item" and "funding item" seem to be the same thing - except the `key`. Could we possibly keep just one?

Reply:
`FundingItem` wasn't even used, my mistake. It's removed.